Tools/Scripts/libraries/webkitapipy/webkitapipy/sdkdb.py

```diff
@@ -466,6 +466,10 @@ def audit(self) -> Iterable[Diagnostic]:
                 # Allowed but also exported => unnecessary allowlist entry to
                 # remove.
                 for path in sorted(set(allowlist_paths.split(','))):
+                    # Normally, a declaration would only be exported from one
+                    # library in the SDK. If the cache sees multiple sources,
+                    # just pick one.
+                    export_path = export_paths.split(',')[0]
```

Comment:
This looks like we do care about reproducible sorting.

Reply:
Good point. We should, otherwise the diagnostic emitted might flip between different paths.